### ISSUE TITLE
ILS frequency not (always) entered automatically

### ISSUE THREAD

**Issue by michjans** [Heavy-Division/B78XH#195]
Hello,
First of all, I like flying the 787 with this mod very much, so thank you for creating this!

Today, I was flying to KLAX. On dep/arr page I had selected the a STAR and approach for ILS07L. But, the frequency for the ILS was not entered automatically and as a result I had no LOC/glideslope indicator on the PFD (and autopilot did not intercept the ILS as a result).
When looking at the NAV/RAD page it did show "ILS 07L/ [ ]" under ILS-MLS, but no frequency.
After I entered the ILS frequency (and CRS) manually, the LOC/glideslope indicator did appear on the PFD and the autopilot started intercepting the ILS.

I didn't notice this problem before. I never had to enter the ILS frequency manually. I am using the latest released version 0.1.13.

Could this be a bug or did I miss something?

Thanks!

**Comment by TDiver1715** [comment]
Hello!  I cannot confirm, but I believe it to be a MSFS bug because I have had it happen over time with multiple aircraft.   In my case, it is totally random; no "common denominator".Perhaps others may have more insight?
-------- Original message --------From: michjans ***@***.***> Date: 2/18/22  09:54  (GMT-05:00) To: Heavy-Division/B78XH ***@***.***> Cc: Subscribed ***@***.***> Subject: [Heavy-Division/B78XH] ILS frequency not (always) entered
  automatically (Issue #195) 
Hello,
First of all, I like flying the 787 with this mod very much, so thank you for creating this!
Today, I was flying to KLAX. On dep/arr page I had selected the a STAR and approach for ILS07L. But, the frequency for the ILS was not entered automatically and as a result I had no LOC/glideslope indicator on the PFD (and autopilot did not intercept the ILS as a result).
When looking at the NAV/RAD page it did show "ILS 07L/ [ ]" under ILS-MLS, but no frequency.
After I entered the ILS frequency (and CRS) manually, the LOC/glideslope indicator did appear on the PFD and the autopilot started intercepting the ILS.
I didn't notice this problem before. I never had to enter the ILS frequency manually. I am using the latest released version 0.1.13.
Could this be a bug or did I miss something?
Thanks!
